utils/releaser/src/FileManipulator/VersionUpgradeFileManipulator.php

```diff
@@ -12,52 +12,55 @@ final class VersionUpgradeFileManipulator
 {
     /**
      * @var string
-     * @see https://regex101.com/r/izBgtv/3
+     * @see https://regex101.com/r/izBgtv/7
      */
-    private const HEADLINE_WITH_LINK_PATTERN = '#\# \[Upgrade from [\w.-]+ to Unreleased\]\(.+\)#';
+    private const HEADLINE_WITH_LINK_PATTERN = '#\# \[Upgrade from [\w.-]+ to [\w.-]+\]\(.+\)#';
 
     /**
      * @var string
      */
-    private const FILE_CONTENT_INFORMATION_PATTERN = '#This guide contains instructions to upgrade from version .* to Unreleased#';
+    private const FILE_CONTENT_INFORMATION_PATTERN = '#This guide contains instructions to upgrade from version .* to .*#';
```

Comment:
Do we really need the `.*` at the end? We know that it should end with `-dev` right?

Reply:
`-dev` added :slightly_smiling_face: